util/design/lib/OtpMemMap.py

```diff
@@ -206,8 +207,19 @@ def _validate_mmap(config):
                 "inv_default": "<random>"
             })
             # Randomize the digest default.
-            random_or_hexvalue(part["items"][-1], "inv_default", DIGEST_SIZE * 8)
+            random_or_hexvalue(part["items"][-1], "inv_default",
+                               DIGEST_SIZE * 8)
+
+            if offset > check_int(part["offset"]) + check_int(
+                    part["size"]) - DIGEST_SIZE:
```

Comment:
Oof, the line wrapping makes this quite hard to read. Maybe define a local variable for the right hand side? (You also use it again later to update offset)

Reply:
good point - fixed!